aybapi/utils/errors.py

```diff
@@ -0,0 +1,3 @@
+class APIError(Exception):
```

Comment:
Should this be RuntimeException?

Also what are you returning the status code in if you got a 500 internal server error, or 404 not found?